catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloader.java

```diff
@@ -487,8 +494,10 @@ public void run() {
               downloadIdentifier);
         }
       }
-    } catch (IOException e) {
+    } catch (IOException | InterruptedException e) {
```

Comment:
❗️  I'm not sure we should be sharing a catch here if we're going to be raising the interrupt flag. I'd suggest breaking these apart